sensor/kubernetes/selector/selector.go

```diff
@@ -66,31 +72,32 @@ func (d selectorDisjunction) Matches(labels labelsWithLen) bool {
 	return false
 }
 
-// or returns the logical or of the given SelectorWrappers.
-func or(sels ...selector) selector {
+// Or returns the logical or of the given SelectorWrappers.
+func Or(sels ...Selector) Selector {
 	return selectorDisjunction(sels)
 }
 
-type selectorWrapOption func(*selectorWrap)
+// WrapOption function interface to define properties of selectors
+type WrapOption func(*Wrap)
 
-// emptyMatchesNothing means that a set with no labels should not match with anything
-func emptyMatchesNothing() selectorWrapOption {
-	return func(sw *selectorWrap) {
+// EmptyMatchesNothing means that a set with no labels should not match with anything
+func EmptyMatchesNothing() WrapOption {
+	return func(sw *Wrap) {
 		sw.matchNil = false
 	}
 }
 
-// emptyMatchesEverything means that a set with no labels should match with everything
-func emptyMatchesEverything() selectorWrapOption {
-	return func(sw *selectorWrap) {
+// EmptyMatchesEverything means that a set with no labels should match with everything
+func EmptyMatchesEverything() WrapOption {
+	return func(sw *Wrap) {
 		sw.matchNil = true
 	}
 }
 
-// createSelector returns a SelectorWrapper for the given map of labels; matchNil determines whether
+// CreateSelector returns a SelectorWrapper for the given map of labels; matchNil determines whether
 // an empty set of labels matches everything or nothing.
-func createSelector(labelsMap map[string]string, opts ...selectorWrapOption) selectorWrap {
-	selWrapper := selectorWrap{matchNil: false}
+func CreateSelector(labelsMap map[string]string, opts ...WrapOption) Wrap {
```

Comment:
[nit] I think we could return the `Selector` interface here.

Reply:
Done